lib/src/Base/Geom/LevelSet.cxx

```diff
@@ -125,6 +128,50 @@ void LevelSet::setLevel(const NumericalScalar level)
   level_ = level;
 }
 
+/* Lower bound of the bounding box */
+NumericalPoint LevelSet::getLowerBound() const
+{
+  const UnsignedInteger dimension(getDimension());
+  NumericalPoint lowerBound(dimension);
+  LinearNumericalMathFunction translate(NumericalPoint(1, level_), NumericalPoint(1), IdentityMatrix(1));
+  NumericalMathFunction equality(translate, function_);
+  for (UnsignedInteger i = 0; i < dimension; ++i)
+    {
+      Matrix m(1, dimension);
+      m(0, i) = 1.0;
+      LinearNumericalMathFunction coordinate(NumericalPoint(dimension), NumericalPoint(1), m);
+      OptimizationProblem problem(coordinate, equality, NumericalMathFunction(), Interval());
+      problem.setMinimization(true);
+      Cobyla solver(problem);
```

Comment:
Cobyla is hardcoded here, but this class should have a setOptimizationSolver to let users override this choice, shouldn't it?